.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,44 @@
+name: Pygmalion UI Front End - Build / Quality Control
+
+permissions:
+  contents: read # This is required for actions/checkout
+
+on:
+  workflow_dispatch:   # Allows you to run this workflow manually from the Actions tab
+
+jobs:
+  build:
+    name: Pipeline
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Install Performant Node Package Manager (NPM)
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.2.1
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.4.0'
+          cache: 'pnpm'
+
+      - name: Cache Dependencies
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/pnpm-lock.yaml') }}
```

Comment:
What does `cache: 'pnpm'` do? If it already takes care of caching the dependencies that PNPM installs, do we need the extra cache action? I'm curious because the [GitHub Actions example for pnpm](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#example-caching-dependencies) doesn't seem to include the cache action.

Reply:
You're right, this appears to be redundant caching. This code was modified from a previous project using npm which didn't include the second caching method

Reply:
Got it. If you can confirm that, this seems worth dropping then.